5001-6000/LIT5673AkkonuColl.xml

```diff
@@ -209,12 +211,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
 
                 <div type="textpart" xml:id="AkkColl6">
-                    <label>Miracle of the monk Isaac</label>
-
-                    <div type="textpart" xml:id="AkkMir6" corresp="LIT3592Miracle">
-                        <label>Miracle</label>
+                    <label>Miracle of the monk Yǝsḥaq</label>
+                    <div type="textpart" xml:id="AkkMir6A" corresp="LIT3592Miracle">
+                        <label>The Revelation of the Virgin Mary to the Devoted Monk Yǝsḥaq</label>
+                    </div>
+                    <div type="textpart" xml:id="AkkMir6B" corresp="LIT3592Miracle">
```

Comment:
here you must restore the reference to AkkMir6 corresponding to LIT3592 as it used to be, revert your changes as already remarked